apps/passport-client/src/loadInitialState.ts

```diff
@@ -43,20 +44,25 @@ export async function loadInitialState(): Promise<AppState> {
     self &&
     (!self.semaphore_v4_commitment || !self.semaphore_v4_pubkey)
   ) {
-    const semaphoreV4IdentityPCD = findIdentityV4PCD(pcds);
-    if (semaphoreV4IdentityPCD) {
-      await requestUpgradeUserWithV4Commitment(
-        appConfig.zupassServer,
-        await makeUpgradeUserWithV4CommitmentRequest(pcds)
-      );
-    }
+    const identityPCD = v3IdentityToPCD(identityV3);
+
+    await requestUpgradeUserWithV4Commitment(
```

Comment:
What about the return value of this?  It may be okay to ignore if this fails, but it would be good to explain in a comment why it's appropriate.

Reply:
I couldn't think of a productive fallback, i'll document that in a comment,